config/environments/production.rb

```diff
@@ -24,7 +24,7 @@
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
   # Compress JavaScripts and CSS.
-  config.assets.js_compressor = :uglifier
+  config.assets.js_compressor = :terser
```

Comment:
encountered https://github.com/lautis/uglifier/issues/185

Reply:
Wow, so it was because our JS is ES6? Haha can't believe the FE was so advanced.